tools/crm_mon.c

```diff
@@ -385,9 +399,22 @@ get_option_desc(char c)
             continue;
 
         if (long_options[lpc].val == c) {
-            const char * tab = NULL;
-            tab = strrchr(long_options[lpc].desc, '\t');
-            return tab ? ++tab : long_options[lpc].desc;
+            static char *buf = NULL;
+            const char *rv;
+            char *nl;
+
+            /* chop off tabs and cut at newline */
+            free(buf);
```

Comment:
With this approach we leak the last description shown. I'm thinking we should just merge print_option_help() and get_option_desc() so the memory management can be more natural.

Reply:
To be honnest I don't like this kind of code either but why would we leak anything?
The memory from the previous use would just be kept and freed when it is replaced by the new one.